### ISSUE TITLE
Multiple selects are broken for array columns

### ISSUE THREAD

**Issue by StareIntoTheBeard** [jakub-zawislak/formex_ecto#5]
When a schema has an array column, the multiple select does not need to be cast to associated schema. You should be able to save multiple values back to an array column. From what I can tell, the bug is in `lib/changeset.ex #cast_multiple_selects`.

**Comment by jakub-zawislak** [comment]
Could you give some sample code? I have never used array columns in Postgres. I have tried to use it in this way:

```elixir
def change do
  alter table(:users) do
    add :degrees, {:array, :string}
  end
  execute "UPDATE users SET degrees = '{}'"
end
```

```elixir
schema "users" do
  field :degrees, {:array, :string}
end
```

```elixir
form
|> add(:degrees, :multiple_select, choices: [
  "Bachelor",
  "Master",
  "Doctor",
])
```

~~Using code from your pull request #6, I'm having this error on submit: `function nil.related/0 is undefined or private` in `lib/changeset.ex:52`~~

Edit: It works, my code just didn't recompiled after pull. I will merge that pull request soon.

**Comment by StareIntoTheBeard** [comment]
Ah, no worries. Was thinking maybe that was the case.  At any rate, your code above is how I'm using it in our project. 
https://github.com/gitcoinco/codefund/blob/master/lib/code_fund_web/forms/user_type.ex#L15
Thank you!

**Comment by jakub-zawislak** [comment]
Fixed in formex_ecto `0.1.9` and formex `0.6.5` (in the main lib there was a problem when field value was `nil`)
